packages/server-wallet/src/db/migrations/20201014174039_open-channel-objectives.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+const tableName = 'open-channel-objectives';
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable(tableName, function(table) {
```

Comment:
It seems like none of these columns should be allowed to have a `NULL` value? If so, they should be have a `NOT NULL` constraint.